src/routes/user/user.routes.js

```diff
@@ -5,6 +5,6 @@ import userMiddlewares from '../../middlewares/user/user.middlewares.js';
 const router = Router();
 
 router.get('/dashboard', userMiddlewares.isloggedIn, userMiddlewares.emailVerificationMiddleware, userControllers.getUserDashboard);
-router.post('/dashboard', userMiddlewares.isloggedIn, userControllers.addVehicle);
+router.post('/dashboard/addVehicle', userMiddlewares.isloggedIn, userControllers.addVehicle);
```

Comment:
Don't use upper-casing in url-path (except tokens and others)

Reply:
so which one is preferable?
/add-vehicle or /addvehicle